src/jsutils.h

```diff
@@ -59,12 +59,28 @@ char *flash_strncpy(char *dest, const char *source, size_t cap);
     and extract just the 8 bits we want */
 #define READ_FLASH_UINT8(ptr) ({ uint32_t __p = (uint32_t)(char*)(ptr); volatile uint32_t __d = *(uint32_t*)(__p & (uint32_t)~3); ((uint8_t*)&__d)[__p & 3]; })
 
+#define __concat(o,l) __to_rom##l
+#define jsvObjectSetChildAndUnLock(obj,name,child) jsvObjectSetChildAndUnLock_FLASH_CTR(obj,__COUNTER__,name,child)
```

Comment:
I don't understand why this is needed. What is the path from the strings in flash to here? Why are the changes to the JswSymPtrs not all contained in the generated jswrapper.c file?